published/Daily_goals_and_results/README

```diff
@@ -0,0 +1 @@
+Includes actions to ask a user about their goals for the day, follow up later to find out how it went, and to report on the whole team.
```

Comment:
Suggested edits:

"Ellipsis will ask members of a team about their goals for the day, follow up later to find out how it went, and report everyone's results to the team."